src/views/darkroom.c

```diff
@@ -2221,6 +2224,21 @@ void init_key_accels(dt_view_t *self)
 
   // fullscreen view
   dt_accel_register_view(self, NC_("accel", "full preview"), GDK_KEY_z, 0);
+
+  // undo/redo
+  dt_accel_register_view(self, NC_("accel", "undo"), GDK_KEY_z, GDK_CONTROL_MASK);
+  dt_accel_register_view(self, NC_("accel", "redo"), GDK_KEY_r, GDK_CONTROL_MASK);
+
+}
+
+static gboolean _darkroom_undo_callback(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
+                                        GdkModifierType modifier, gpointer data)
+{
+  if(keyval == GDK_KEY_z)
```

Comment:
So now that ^ accel user can change will be ignored and default value will be just assumed?